app.py

```
@@ -17,8 +17,17 @@
 tempo = None  # Global variable for tempo
 
+# Directories for uploads
+UPLOADS_FOLDER = 'uploads'
+TEMPORARY_UPLOADS_FOLDER = 'temporary_uploads'
+
+# Ensure directories exist
+os.makedirs(UPLOADS_FOLDER, exist_ok=True)
+os.makedirs(TEMPORARY_UPLOADS_FOLDER, exist_ok=True)
+
 @app.after_request
 def add_header(response):
```

Ruff (0.8.0)

Missing return type annotation for public function `add_header`

(ANN201)